portfolio/src/main/webapp/script.js

```diff
@@ -13,13 +13,20 @@
 // limitations under the License.
 
 /**
- * Fetches a welcome messgae to the home page.
+ * Fetches comments and appends them as list elements.
  */
-function sayWelcome() {
-  fetch('/data').then(response => response.text()).then((text) => {
-    document.getElementById('home-welcome').innerText = text;
+function getComments() {
+  fetch('/data').then(response => response.text()).then((comments) => {
+    const comments_obj = JSON.parse(comments);
```

Comment:
Google's variable naming rules would make this `commentsObj`.

Reply:
See also https://en.wikipedia.org/wiki/Camel_case#:~:text=Camel%20case%20(stylized%20as%20camelCase,no%20intervening%20spaces%20or%20punctuation. :)